omod/src/main/resources/config.xml

```diff
@@ -59,6 +59,13 @@
         </description>
     </globalProperty>
 
+    <globalProperty>
+        <property>patientmatching.unknownPatientAttributeTypeUuid</property>
+        <description>The uuid of the person attribute type that is used to mark a patient as unknown, this assumes the
```

Comment:
What does `where true is the value for the unknown patient` mean?

Reply:
Meaning if the person attribute value is set true

Reply:
Do you mind making it clearer?

Reply:
Done